src/mesh/Router.cpp

```diff
@@ -469,8 +469,15 @@ void Router::perhapsHandleReceived(meshtastic_MeshPacket *p)
     // assert(radioConfig.has_preferences);
     bool ignore = is_in_repeated(config.lora.ignore_incoming, p->from) || (config.lora.ignore_mqtt && p->via_mqtt);
 
+    // Neighbor info module is disabled, ignore expensive neighbor info packets
+    if (p->which_payload_variant == meshtastic_MeshPacket_decoded_tag &&
```

Comment:
Most of the times this packet will not yet be decrypted here, because it will arrive here directly after: https://github.com/meshtastic/firmware/blob/a58348369d0ffeaef28bd585c89a649ee7840d93/src/mesh/RadioLibInterface.cpp#L379
which puts it in the `fromRadioQueue`.

Reply:
I see. How about cancelling in Router's handleReceived after the payload is decoded?

Reply:
Sounds good.